Add SSL support for backend connections in PGSQL monitor

rahim-kanji opened the pull request:
PostgreSQL monitor now **supports client certificates** for SSL/TLS connections to backend servers.

* Added two new metrics,` PgSQL_Monitor_ssl_connections_OK` and `PgSQL_Monitor_non_ssl_connections_OK` (in `stats_pgsql_global`), to improve visibility into PostgreSQL monitor connection status.

Closes #[5205](https://github.com/sysown/proxysql/issues/5205)

rahim-kanji committed:
Add SSL support for backend connections in PGSQL monitor

renecannao commented:
@rahim-kanji : we should add some TAP test  , at least to verify that SSL is being used for monitoring

rahim-kanji committed:
Add SSL and non-SSL connection OK metrics for PostgreSQL monitor connections
Adds two new metrics, ssl_connections_OK and non_ssl_connections_OK, to improve visibility into PostgreSQL monitor connection status.

rahim-kanji committed:
Added TAP test

rahim-kanji committed:
Added TAP test to groups.json

rahim-kanji commented:
retest this please
